proxy-router/internal/interfaces/validation.go

```diff
@@ -0,0 +1,11 @@
+package interfaces
+
+import (
+	"context"
+	"math/big"
+	"time"
+)
+
+type Validation interface {
```

Comment:
Same as above,  `EthConnectionValidator` makes more sense to me.

Also, move this interface to the package that is using it (system).